packages/astro/components/Generator.astro

```diff
@@ -0,0 +1 @@
+<meta name="generator" content={Astro.generator} />
```

Comment:
in the interest of a smaller core, is `Astro.generator` enough to accomplish this goal? The dedicated component feels like overkill, and when I hear "Generator component" I immediately thought of some advanced Astro concept. 

Having this snippet example available somewhere in our docs feels like 90% of the value!

Reply:
Agreed! I'm in favor of only adding `Astro.generator` and putting this snippet in our docs and examples.

Reply:
Removed `Generator` in [a07f316](https://github.com/withastro/astro/pull/4012/commits/a07f31673982ebd0812f532fccfc8b406937e674)